### DISCUSSION TITLE
Add minimap icons for Fire pits from Making Friends with My Arm

### DISCUSSION THREAD

**Discussion by Gamer1120** [runelite/runelite#15359] [Ideas]
See title.

**Comment by sirlaige** [comment]
How would this be helpful?

Edit:  What I mean is that the map is useful all the time but these fire-pits only need to be found once and lit.  After you are done with em they would just take up space on the map.  I could be wrong tho, is it useful to have them on the map all the time?

I would like to actually remove some icons, like water and cooking ranges.  To me these ones just take up space as they aren't useful to me.

**Comment by areebahmed04** [comment]
Hi @abextm Can I start working on this?
